view/frontend/web/internals/common.js

```diff
@@ -566,9 +566,9 @@ requirejs(['algoliaBundle'], function(algoliaBundle) {
 					const portWithPrefix = port === '' ? '' : `:${port}`;
 					// IE <= 11 has no location.origin or buggy. Therefore we don't rely on it
 					if (!routeState || Object.keys(routeState).length === 0)
-						return `${protocol}//${hostname}${portWithPrefix}${pathname}`;
+                        return protocol + '//' + hostname + portWithPrefix + pathname;
```

Comment:
In JS files we use tabs instead of spaces (I know, there is an inconsistency), could you fix that? 🙏

Reply:
tabs tabs tabs!